src/devices/video/mc6845.cpp

```diff
@@ -417,8 +422,23 @@ void mc6845_device::recompute_parameters(bool postload)
 	uint16_t vert_pix_total = (m_vert_char_total + 1) * video_char_height + m_vert_total_adj;
 
 	/* determine the visible area, avoid division by 0 */
-	uint16_t max_visible_x = m_horiz_disp * m_hpixels_per_column - 1;
-	uint16_t max_visible_y = m_vert_disp * video_char_height - 1;
+	uint16_t visible_width = m_horiz_disp * m_hpixels_per_column;
+	uint16_t visible_height = m_vert_disp * video_char_height;
```

Comment:
If you’re maintaining the visible area, you want to start with `m_visible_width` and `m_visible_height` here to avoid getting a zero in the first place.